docs/sources/alerting/alerting-rules/declare-incident-from-alert.md

```diff
@@ -0,0 +1,44 @@
+---
+aliases:
+description: Declare an incident from a firing alert
+keywords:
+  - grafana
+  - alert rules
+  - incident
+title: Declare an incident from a firing alert
+weight: 430
+---
+
+# Declare an incident from a firing alert
+
+Declare an incident from a firing alert to streamline your alert to incident workflow.
+
+## Before you begin
+
+- Ensure you have Grafana Incident installed
+- You must have a firing alert
+
+## Procedure
+
+To declare an incident from a firing alert, complete the following steps.
+
+1. Navigate to Alerts & Incidents -> Alerting -> Alert rules.
```

Comment:
Additionally the button is also available from the detail page of an alert, in case a user ends up there via a notification.

Should we include that in the docs?